test/inspector/test-async-stack-traces-promise-then.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const common = require('../common');
+common.skipIfInspectorDisabled();
+const helper = require('./inspector-helper');
+const assert = require('assert');
+
+const script = 'Promise.resolve().then(() => { debugger; });';
```

Comment:
And can we add a test where we create one promise and then have two different chained callback:
function foo() { debugger; }
var p = Promise.resolve();
p.then(foo);
p.then(foo);
And check that we have correct stacks on both breaks.